content/source/docs/enterprise/before-installing/network-requirements.html.md

```diff
@@ -89,7 +89,13 @@ When [Cost Estimation](/docs/enterprise/admin/integration.html#cost-estimation-i
 
 ## Other Configuration
 
-1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example, to do this with UFW run: `ufw allow in on docker0`. This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.
+1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. 
+   * To use UFW, run: `ufw allow in on docker0`.
+   *  To use firewalld, run:
+
+        `firewall-cmd --permanent --zone=trusted --change-interface=docker0`
+
+    This rule can be added before the `docker0` interface exists, so we recommend adding it before installing Docker.
```

Comment:
One more tiny edit now that I see it all together - I promise I'm done after this :-) 

```suggestion
1. If a firewall is configured on the instance, run one of the following to allow traffic to flow out of the `docker0` interface to the instance's primary address. We recommend doing this before you install Docker.
   * To use UFW, run: `ufw allow in on docker0`
   *  To use firewalld, run: `firewall-cmd --permanent --zone=trusted --change-interface=docker0`
```

Reply:
No worries. As long as it becomes the best version of itself :)